docs/en/changes/changes.md

```diff
@@ -12,9 +12,12 @@
 
 #### UI
 
-* Fix: tab active incorrectly, when click tab space 
+* Fix: tab active incorrectly, when click tab space
 * Add impala icon for impala JDBC Java agent plugin.
 * (Webapp)Bump up snakeyaml to 1.31 for fixing CVE-2022-25857
+* [Breaking Change]: migrate from Spring Web to Armeria, now you should use the environment variable name `SW_OAP_ADDRESS`
```

Comment:
How about we use codes to check and remove http:// prefix if exist?

This breaking change seems not necessary.

Reply:
This is how I did currently. This line didn't mention the http prefix.